src/index.ts

```
@@ -1,6 +1,7 @@
 import axios from "axios";
 import { ethers } from "ethers";
 import { ChainId } from "sushi";
+import { RpcState } from "./rpc";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)